webpage/_data/navigation.yml

```diff
@@ -13,3 +13,9 @@ site_nav:
     children:
       - title: "Installing the IOOS conda environment"
         url: /other_resources
+      - title: "Australian ocean data network portal user guide"
```

Comment:
I'm concerned folks won't know what AODN is. AODN is part of IMOS - another little know acronym for the Integrated Marine Observing System.  I like using the title "IMOS User Code Library" since it's shorter and that is really what the site gives access to.